content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,51 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## `istioctl` auto-completion
+
+If you are using Bash or Zsh, `istioctl` includes a tab completion file that you can use. 
+
+### istioctl on Linux
+
+Both the Bash and Zsh completion files are located in the `tools` directory. Copy the preferred completion file to your home directory, and add the following line to source the `istioctl` completion file in your `.bashrc` file. For example with Bash, you will be adding the following line:
```

Comment:
would be good to provide exact cmd so users can copy-n-paste.

Reply:
I tried to explain this as "agnostic" as possible. @ericvn does the tab completion for Zsh work the same way? If so, I can use more direct/exact language here.

Reply:
Copy file to /usr/local/etc/bash_completion.d/istioctl as option to not have to source the file

Reply:
Since zsh isn't ready yet, we're going to omit the zsh copy for now.